frontend/src/shared/components/ConfirmationDialog.tsx

```diff
@@ -23,7 +23,7 @@ const ConfirmationDialog = ({
   };
   return (
     <Dialog {...props}>
-      <DialogTitle>Are you sure?</DialogTitle>
```

Comment:
zostaw to tak jak było albo zrób parametr na tytuł domyślną wartością "Are you sure?"

Reply:
Przywrócę za chwilę